pkg/gitops/templates_test.go

```diff
@@ -98,15 +98,15 @@ var renderAllFilesCases = map[string]struct {
 			"Chart.yaml":  renderedChartYAML,
 		},
 	},
-	"an unused template variable is present (it's ignored)": {
+	"an unused template variable is present (its ignored)": {
```

Comment:
nit: in this case it should actually stay as `it's` . More info: https://www.grammarly.com/blog/its-vs-its/